goat/models.py

```diff
@@ -28,17 +26,16 @@ class Authority(BasicAccessionMixin):
     name = models.CharField(max_length=255)
     namespace = models.CharField(max_length=255, **opt)
     description = models.TextField(**opt)
-
-    configuration = models.TextField(**opt)
-    """JSON-serialized configuration (if available) for this authority."""
-
     builtin_identity_system = models.ForeignKey('IdentitySystem', on_delete=models.CASCADE, **opt)
 
     @property
     def manager(self):
-        if not self.configuration:
-            raise AttributeError("Configuration unavailable for %s" % self.name)
-        return AuthorityManager(self.configuration)
+        if self.builtin_identity_system.name == 'builtin:Conceptpower':
+            from concepts.conceptpower import ConceptPower
+            return ConceptPower()
+        if self.builtin_identity_system.name == 'builtin:VIAF':
+            from concepts.viaf import Viaf
+            return Viaf()
```

Comment:
If we add another identity system this will have to be updated right? If so is that documented?

Reply:
Yes. We need to write a whole class similar to one of these for adding another type of identity system. I updated it in the documentation https://diging.atlassian.net/wiki/spaces/VGNWB/pages/2086273044/Vogon+Initial+Setup